src/http.rs

```diff
@@ -649,16 +645,12 @@ pub fn handle_ui_asset_request(
 }
 
 pub fn send_ws_push(
-    node: String,
     channel_id: u32,
     message_type: WsMessageType,
     blob: KiBlob,
 ) -> anyhow::Result<()> {
     KiRequest::new()
-        .target(Address::new(
-            node,
-            "http_server:distro:sys".parse::<ProcessId>().unwrap(),
-        ))
+        .target("our@http_server:distro:sys".parse::<Address>()?)
```

Comment:
just for performance and cleanness' sake i'd prefer to use the syntax
```
.target(("our", "http_server", "distro", "sys"))
```
wherever possible